Src/Plugins/Security/SEC101_054.AlibabaAccessKeyValidator.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+// From the Nuget package
+using Aliyun.Acs.Core;
+using Aliyun.Acs.Core.Exceptions;
+using Aliyun.Acs.Core.Profile;
+
+// From the DLL
```

Comment:
Delete these comments. They aren't necessary: you can always hit F12 on any reference and discover the binary that holds the definition of the type.

Reply:
Are you sure?  When I do this, I get the error message, "Cannot navigate to the symbol under the caret"

---
In reply to: [591882198](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591882198) [](ancestors = 591882198)

Reply:
Done.

---
In reply to: [591915798](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591915798) [](ancestors = 591915798,591882198)